lib/ui/home/home_page.dart

```diff
@@ -57,9 +61,17 @@ class _HomePageState extends State<HomePage> {
     });
   }
 
+  void getUserData() async{
+    final user = context.select((AuthenticationBloc bloc) => bloc.state.user);
+    Uri uri = Uri.parse('http://3.211.168.157:8000/users/${user.id}');
+    Response response = await get(uri);
+    userData =  jsonDecode(response.body);
+  }
+
```

Comment:
Why are we using a raondom Uri instead of the Authentication bloc?

If you want to do something with User stuff, don't do that in the Home Page. 

HomePage is "Presentation Layer", it should have anything related to "Business Logic" or "Data" layers